tests/test_export.py

```diff
@@ -0,0 +1,43 @@
+'''Test NWB export function
+'''
+
+__all__ = ['dj_config', 'pipeline', 'lab_csv', 'lab_project_csv', 'lab_user_csv',
+           'lab_publications_csv', 'lab_keywords_csv', 'lab_protocol_csv',
+           'lab_project_users_csv', 'ingest_lab', 'subjects_csv', 'subjects_part_csv',
+           'ingest_subjects', 'sessions_csv', 'ingest_sessions']
+
+from . import (dj_config, pipeline, lab_csv, lab_project_csv, lab_user_csv, 
+               lab_publications_csv, lab_keywords_csv, lab_protocol_csv,
+               lab_project_users_csv, ingest_lab, subjects_csv, subjects_part_csv, 
+               ingest_subjects, sessions_csv, ingest_sessions)
+
+def test_session_to_nwb(pipeline, lab_csv, lab_project_csv, lab_user_csv,
+                         lab_publications_csv, lab_keywords_csv, lab_protocol_csv, 
+                         lab_project_users_csv, ingest_lab, subjects_csv, 
+                         subjects_part_csv, ingest_subjects, sessions_csv, 
+                         ingest_sessions):
```

Comment:
Is it the case that all pytest fixtures need to be called, or is `lab_csv` called recursively by `ingest_lab`? This is the same approach as I've taken, calling all sub-fixtures, but it occurs to me now that it may not be required

Reply:
I just did a quick test and it looks like all pytest fixtures need to be passed into the function.